components/Layout.tsx

```diff
@@ -289,14 +290,30 @@ const Footer = () => (
       <a href='https://github.com/json-schema-org/community/discussions' className='text-sm text-slate-400 hover:text-slate-500 pt-3'>GitHub Community Discussions</a>
       <a href='https://groups.google.com/g/json-schema' className='text-sm text-slate-400 hover:text-slate-500 pt-3'>Google Groups</a>
     </div>
-    <div className='flex flex-col items-start mt-8 sm:mt-16 lg:mt-0'>
+    <div className='flex flex-col items-start mt-8 sm:mt-8 lg:mt-0'>
       <Logo />
       <a href='https://opencollective.com/json-schema' className='text-sm text-slate-400 hover:text-slate-500 pt-3 mt-2'>Open Collective</a>
       <a href='/understanding-json-schema/credits' className='text-sm text-slate-400 hover:text-slate-500 pt-3'>Acknowledgments</a>
     </div>
   </footer>
 )
 
+const OpenJS = () => (
+  <div className={classnames(responsiveClasses, 'mx-auto flex flex-flow grid grid-cols-1 sm:grid-cols-2 gap-2')}>
+    <div className='relative flex flex-col items-start col-start-1 md:inline lg:inline sm:hidden hidden'>
+      <div className='absolute text-sm bottom-0 mb-8'>© Copyright JSON Schema Organisation 2023</div>
+    </div>
+    <div className='flex flex-col items-start col-start-2 md:col-start-2 md:col-span-1 sm:col-start-1 sm:col-span-2'>
+      <img className='h-16 mb-3.5' src="/img/logos/openjs_foundation-logo-horizontal-color.svg" alt="color openjs foundation logo"></img>
```

Comment:
I am getting some error when executing the build command of the site and seems to be related to the usage of double quotes:

```
Failed to compile.

./components/Layout.tsx
307:40  Error: Unexpected usage of doublequote.  jsx-quotes
307:101  Error: Unexpected usage of doublequote.  jsx-quotes
308:20  Error: Unexpected usage of doublequote.  jsx-quotes
308:99  Error: Unexpected usage of doublequote.  jsx-quotes
308:252  Error: Unexpected usage of doublequote.  jsx-quotes
308:430  Error: Unexpected usage of doublequote.  jsx-quotes
308:544  Error: Unexpected usage of doublequote.  jsx-quotes
308:662  Error: Unexpected usage of doublequote.  jsx-quotes
308:816  Error: Unexpected usage of doublequote.  jsx-quotes
309:20  Error: Unexpected usage of doublequote.  jsx-quotes
309:97  Error: Unexpected usage of doublequote.  jsx-quotes
309:201  Error: Unexpected usage of doublequote.  jsx-quotes
309:309  Error: Unexpected usage of doublequote.  jsx-quotes
309:421  Error: Unexpected usage of doublequote.  jsx-quotes
309:517  Error: Unexpected usage of doublequote.  jsx-quotes
309:631  Error: Unexpected usage of doublequote.  jsx-quotes
309:747  Error: Unexpected usage of doublequote.  jsx-quotes
309:859  Error: Unexpected usage of doublequote.  jsx-quotes

info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Could it be possible to use single quotes to see if that fix the message?

Reply:
Pushed a commit removing double quotes and replacing with singel quotes.